packages/components/src/alignment-matrix-control/utils.js

```diff
@@ -11,15 +11,15 @@ export const GRID = [
 
 // Stored as map as i18n __() only accepts strings (not variables)
 export const ALIGNMENT_LABEL = {
-	'top left': __( 'Top Left' ),
+	'top left': ! isRTL() ? __( 'Top Left' ) : __( 'Top Right' ),
```

Comment:
Responding to [this](https://github.com/WordPress/gutenberg/issues/36237#issuecomment-1210688191):

> Note: In my environment, `isRTL` function did not respond correctly when I changed to the RTL via RTL Tester.

I think this is because the `isRTL` call here is being made in an object that is completely outside of the AlignmentMatrixControl component function. So the values becomes static when the JS bundle is loaded. For the Storybook RTL switcher to work, the `isRTL` needs to be called again when the AlignmentMatrixControl component is re-rendered. Could you try reworking it so the conditional fires again when the component is re-rendered? For example, using a `getAlignmentLabel()` function instead of accessing an [object](https://github.com/WordPress/gutenberg/blob/8a22f96709b4f6172a58e4a5de88c26576beb683/packages/components/src/alignment-matrix-control/cell.js#L18).

Hope that makes sense. It doesn't matter in actual WP usage, but it's helpful for it to be testable in Storybook if possible 🙂

Reply:
It's working well now, thanks 🎉